app/views/layouts/application.html.erb

```diff
@@ -25,7 +26,7 @@
           {name: 'Help', component: 'Help', href: '/help'},
           {name: 'Logout', component: 'Logout', href: '/sign_out'}
         <% end %>
-      ]"/>
+      ]"></menu-bar>
```

Comment:
This seems to be solving something else

Reply:
According to the console errors, IE11 requires full closing tags. Only the header renders without this fix.